### DISCUSSION TITLE
Can't run Graphcast through getting started

### DISCUSSION THREAD

**Discussion by Jerry0424** [NVIDIA/earth2mip#169] [Q&A] [answered]
I try to run the basic graphcast model using Getting Started command but I get the error and have no idea how to solve it. 
![未命名](https://github.com/NVIDIA/earth2mip/assets/71301727/91226f1b-4d29-455e-b005-4dad5a2d5160)

**Comment by nbren12** [comment, marked as answer]
This happens because graphcast wants the total precip accumulated over 6 hours as an input and this is not in CDS. I have a pr fixing this here https://github.com/NVIDIA/earth2mip/pull/94, but didn’t like the implementation.

for now you can use the graphcast_operational model, which doesn’t have this precip variable as an input.
